packages/backend/src/routers/projectRouter.ts

```diff
@@ -353,6 +355,17 @@ projectRouter.post(
         };
 
         try {
+            const { organizationUuid } = req.user!;
```

Comment:
I created a new ticket to move all this logic into a service https://github.com/lightdash/lightdash/issues/4900